pkg/crc/api/handlers.go

```diff
@@ -134,28 +134,18 @@ func (h *Handler) SetConfig(args json.RawMessage) string {
 		})
 	}
 
-	var a = make(map[string]interface{})
-
-	err := json.Unmarshal(args, &a)
-	if err != nil {
+	var req client.SetConfigRequest
+	if err := json.Unmarshal(args, &req); err != nil {
 		return encodeStructToJSON(client.SetOrUnsetConfigResult{
 			Success: false,
 			Error:   err.Error(),
 		})
 	}
 
-	configs, ok := a["properties"].(map[string]interface{})
-	if !ok {
-		return encodeStructToJSON(client.SetOrUnsetConfigResult{
-			Success: false,
-			Error:   "No config keys provided",
-		})
-	}
-
```

Comment:
That's a slight change of behaviour if no config keys are provided, after this commit it will return success/empty set of keys. I don't think this matters, both erroring out, or reporting success is imo fine.